### PR TITLE
CRM-15928

### PR DESCRIPTION
**Pull request by seamuslee001**
- [CRM-15928](https://issues.civicrm.org/jira/browse/CRM-15928)

### CODE REVIEW

**Comment by seamuslee001** [comment]
@xurizaemon @totten @eileenmcnaughton Is this good to be merged?

File: templates/CRM/Admin/Form/Setting/Miscellaneous.tpl [hunk 1]
```diff
@@ -73,9 +73,26 @@
             </td>
           </tr>
         {/foreach}
-
+        <tr class="crm-miscellaneous-form-block-recentItemsMaxCount">
```
Comment (totten) [inline templates/CRM/Admin/Form/Setting/Miscellaneous.tpl]: @seamuslee001 @pradpnayak  What do `recentItems*` have to do with `remote_profile_submissions_allowed`?

Comment (xurizaemon) [inline templates/CRM/Admin/Form/Setting/Miscellaneous.tpl]: Nothing at all, this was a merge conflict I ran into and thought I'd cleared out of the PR I submitted. (Those lines are shown in [this PR's final patch](https://patch-diff.githubusercontent.com/raw/civicrm/civicrm-core/pull/8923.patch) but not changed.)

Comment (xurizaemon) [inline templates/CRM/Admin/Form/Setting/Miscellaneous.tpl]: I lied, it looks like my PR did change those lines also 👎

Comment (seamuslee001) [inline templates/CRM/Admin/Form/Setting/Miscellaneous.tpl]: I think that may have come from Pradeep, I have removed them now

File: CRM/Profile/Page/Router.php [hunk 2]
```diff
@@ -97,11 +97,12 @@ public function run($args = NULL) {
     }
 
     if ($secondArg == 'edit' || $secondArg == 'create') {
+      $allowRemoteSubmit = Civi::settings()->get('remote_profile_submissions');
```
Comment (seamuslee001) [inline CRM/Profile/Page/Router.php]: @totten @xurizaemon The way this was before hiding underneath the If in the next line, i'm sure would have led to a unknown variable $allowRemoteSubmit in the create action

**Comment by seamuslee001** [comment]
I have tested this and works as expected, The only thing that could be improved is the error message as it just currently displays the no cookie error. 

I tested by submitting a profile after setting the setting to null got he no cookie error
Changed setting to yes and profile submission was accepted.

**Comment by totten** [comment]
Recent patches make sense. Passes tests and reviewed by Seamus (and also implicitly by Chris/Pradeep). Like the upgrade message. Merging.

FWIW, I'm also copying the upgrade note to https://docs.google.com/spreadsheets/d/1XSLoTNr6RKUvaydGPhdsQMT6orimub1AO1ZEbEe_Mp8/edit#gid=1219056148 so that it winds up in the final release announcement.

**Pull request merged**